components/AppBar/src/MDCAppBarContainerViewController.m

```diff
@@ -72,4 +77,38 @@ - (UIInterfaceOrientation)preferredInterfaceOrientationForPresentation {
   return self.contentViewController.preferredInterfaceOrientationForPresentation;
 }
 
+#pragma mark - Enabling top layout guide adjustment behavior
+
+- (void)updateTopLayoutGuideBehavior {
+  if (self.topLayoutGuideAdjustmentEnabled) {
+    if ([self isViewLoaded]) {
+      self.contentViewController.view.translatesAutoresizingMaskIntoConstraints = YES;
```

Comment:
Why are we changing this value in the contentVC? Should we inspect instead and only change the constraints (.autoresizingMask) if .translates is YES?  What if they have custom constraints to reposition their view by some adjustment from the header?

Reply:
We entirely own the content view controller's layout. In general, views should not make assumptions about their own layout within the context of its parent.

Reply:
OK. So a view should not give itself a fixed width? How does that work for UISwitch and other "fixed-size" elements?

Reply:
Technically-speaking, if you give such a view a frame then that frame should be respected. Views that deviate from this behavior are the exception rather than the norm.

Reply:
Fair enough. Does the AppBar's contentView actually require a flexible width? Can we get away with just a flexible height?

Reply:
Yes, in order to support landscape orientation.

Reply:
Sorry, I don't understand. Why does the header need to resize the contentView to support landscape? If the view's always 50 points wide, how does Landscape affect that?

Reply:
Why do you feel the view is 50 points wide?

This is a view controller — however the app bar container's view controller is laid out, we want that layout to propagate to the content view controller's view. Thus we "inherit" the container's dimensions by setting a flexible width/height and equating the content view controller's frame to the container view controller view's bounds.

Reply:
We spoke offline and I now understand the AppBarContainerView controller isn't for the use case I had in mind.